src/main/java/io/Output.java

```diff
@@ -0,0 +1,87 @@
+package io;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Output {
+
+    static OutputHandlerImpl handler;
+
+    private Output() {}
+
+    public static OutputHandlerImpl getScreen() {
```

Comment:
purpose of this class is to eliminate the hard dependency on OutputHandlerImpl, but this reintroduces it
use OutputHandler instead